app/assets/javascripts/miq_angular_configuration.js

```diff
@@ -0,0 +1,54 @@
+ManageIQ.angular.app.config([ '$httpProvider', '$stateProvider', '$urlRouterProvider', '$locationProvider', function ($httpProvider, $stateProvider, $urlRouterProvider, $locationProvider) {
+  $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
```

Comment:
Don't do this, please use `miqHttpInject` instead... (and since this is ManageIQ.angular.app, which is already httpInjected, you don't need to do anything but remove this line..)